spec/helpers/widget_helper_spec.rb

```diff
@@ -12,16 +12,19 @@
     end
 
     it 'should return "Awaiting classification" for "waiting_classification' do
-      expect(status_description(@info_request, 'waiting_classification')).to eq('Awaiting classification')
+      expect(status_description(@info_request, 'waiting_classification'))
+        .to eq('Awaiting classification')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.